backend/app/models/user.py

```diff
@@ -19,6 +19,9 @@ class User(db.Model):
         nullable=False,
     )
     email = db.Column(db.String, nullable=False)
+    last_modified = db.deferred(
+        db.Column(db.DateTime, default=func.now(), onupdate=func.now())
```

Comment:
After I researched we should be using `server_default` instead of `default`. Basically this takes the timestamp from the DB instead of the server, which is more reliable.

Also, ensure this is non-nullable: `nullable=False`